src/pages/networks/panels/CreateLocalPeeringPanel.tsx

```diff
@@ -0,0 +1,212 @@
+  const projectOtherLabel = "Manually enter project";
+  const networkOtherLabel = "Manually enter network";
+
+  const localPeeringSchema = Yup.object().shape({
+    targetProject: Yup.string().required("Target project is required"),
+
+    customTargetProject: Yup.string()
+      .nullable()
+      .when("targetProject", {
+        is: (v: string) => v === projectOtherLabel,
+        then: (schema) => schema.required("Please enter a project name"),
+        otherwise: (schema) => schema.notRequired(),
+      }),
+
+    targetNetwork: Yup.string().when("targetProject", {
+      is: (v: string) => v !== projectOtherLabel,
+      then: (schema) => schema.required("Target network is required"),
+      otherwise: (schema) => schema.notRequired(),
+    }),
+
+    customTargetNetwork: Yup.string()
+      .nullable()
+      .when("targetNetwork", {
+        is: (v: string) => v === networkOtherLabel,
+        then: (schema) => schema.required("Please enter a network name"),
+        otherwise: (schema) => schema.notRequired(),
+      }),
+
+    name: Yup.string()
+      .test(
+        ...testDuplicateLocalPeeringName(
+          panelParams.project,
+          network.name,
+          controllerState,
+        ),
+      )
+      .required("Local peering name is required"),
+  });
+
+  const handleSuccess = (peerName: string) => {
+    toastNotify.success(
+      <>
+        Local peering{" "}
+        <ResourceLink
+          type={"peering"}
+          value={peerName}
+          to={`${networkURL}/local-peerings`}
+        />{" "}
+        created for network{" "}
+        <ResourceLink type="network" value={network.name} to={networkURL} />.
+      </>,
+    );
+    closePanel();
+  };
+  const formik = useFormik<LocalPeeringFormValues>({
+    initialValues: {
+      name: "",
+      description: "",
+      targetProject: project?.name || "",
+      targetNetwork: "",
+      customTargetNetwork: "",
+      customTargetProject: "",
+      createMutualPeering: false,
+    },
+    validationSchema: localPeeringSchema,
+    validateOnMount: true,
+    validateOnChange: true,
+    validateOnBlur: true,
+    onSubmit: (values) => {
+      const localPeeringPayload = {
+        name: values.name,
+        description: values.description,
+        target_project:
+          values.targetProject === projectOtherLabel
+            ? values.customTargetProject
+            : values.targetProject,
+        target_network:
+          values.targetNetwork === networkOtherLabel
+            ? values.customTargetNetwork
+            : values.targetNetwork,
+      };
+
+      createNetworkPeer(
+        network.name,
+        panelParams.project,
+        JSON.stringify(localPeeringPayload),
+      )
+        .then(() => {
+          if (formik.values.createMutualPeering) {
+            const mutualPeeringPayload = {
+              name: values.name,
+              description: values.description,
+              panelParams: panelParams.project,
```

Comment:
```suggestion
              target_project: panelParams.project,
```
This is the wrong field name.